transcendence-app/src/chat/chat-room-message-with-user.domain.ts

```diff
@@ -0,0 +1,18 @@
+import { User } from '../user/user.domain';
+import { ChatRoomMessageWithUserEntity } from './infrastructure/db/chat-room-message-with-user.entity';
+
+export class ChatRoomMessageWithUser {
+  id: string;
+  user: User;
+  chatRoomId: string;
```

Comment:
the name of this field is inconsistent with chatroom-message.domain.ts . `chatRoomId` vs `chatId`.

Reply:
I guess the chatId will never exist (individual chats will have a userId-userId relation) but agree to update it to chatRoomId so it's clearer what it refers to

Reply:
Good catch. I've fixed it.
`chatRoomId` is a good option too. For this PR, I preferred to rename it to `chatId` to match the names we already have in the `ChatMember` table.